treeherder/auth/backends.py

```diff
@@ -136,8 +136,11 @@ def authenticate(self, request):
 
             # The Django user session expiration should be set to the token for which the expiry is closer.
             session_expiry_in_ms = min(accesstoken_exp_in_ms, idtoken_exp_in_ms)
+            expires_in = (session_expiry_in_ms - now_in_ms) / 1000
 
-            request.session.set_expiry((session_expiry_in_ms - now_in_ms) / 1000)
+            logger.debug("Updating session to expire in %d hours" % (expires_in / 3600))
```

Comment:
Production only logs warning and above, so this would need to be `logger.warn` for it to appear in papertrail:
https://github.com/mozilla/treeherder/blob/58e6690e645de083f0aa0430b31f02510b57cec2/treeherder/config/settings.py#L165-L168